services/streammanager/kafka/client/client.go

```diff
@@ -73,12 +73,10 @@ func New(network string, addresses []string, conf Config) (*Client, error) {
 		}
 
 		sshConfig := &ssh.ClientConfig{
-			User:    conf.SSHConfig.User,
-			Auth:    []ssh.AuthMethod{ssh.PublicKeys(signer)},
-			Timeout: conf.DialTimeout,
-		}
-		if conf.SSHConfig.AcceptAnyHostKey {
-			sshConfig.HostKeyCallback = ssh.InsecureIgnoreHostKey() // skipcq: GSC-G106
+			User:            conf.SSHConfig.User,
+			Auth:            []ssh.AuthMethod{ssh.PublicKeys(signer)},
+			Timeout:         conf.DialTimeout,
+			HostKeyCallback: ssh.InsecureIgnoreHostKey(), // skipcq: GSC-G106
```

Comment:
@saurav-malani as discussed over standup this is an important change I introduced (cc @lvrach) and it's going to apply to all SSH connections for kafka destinations.